cmd/gotpm/main.go

```diff
@@ -2,12 +2,42 @@
 package main
 
 import (
+	"fmt"
 	"os"
+	"runtime/debug"
 
 	"github.com/google/go-tpm-tools/cmd"
 )
 
+// GoReleaser will populates those fields
+var (
+	version = "dev"
+	commit  = "none"
+	date    = "unknown"
```

Comment:
How does goreleaser populate these fields?

Reply:
when building the binary, using ldflags:
https://goreleaser.com/cookbooks/using-main.version/

Reply:
consider adding a comment